### PR TITLE
Document oncall rotation participation.

### PR DESCRIPTION
**Pull request by tcnghia**
Part of https://github.com/knative/serving/issues/8005 .

### CODE REVIEW

File: ROLES.md [hunk 1]
```diff
@@ -214,6 +214,10 @@ approver in an OWNERS file:
 
 * Approve code contributions for acceptance.
 
+Approvers are also expected to participate in community contact rotations to
+support users and keep test quality high, as well as release leads rotation to
+shepherd Knative releases.
```
Comment (mattmoor) [inline ROLES.md:219]: link to the docs about it?

Comment (tcnghia) [reply ROLES.md:224]: Done

**Comment by pmorie** [comment]
apologies for delay

/approve

**Comment by fejta** [comment]
woo

**Pull request merged**